src/runmode-af-packet.c

```diff
@@ -383,13 +383,23 @@ static void *ParseAFPConfig(const char *iface)
                   ebpf_file);
 #endif
         aconf->ebpf_lb_file = ebpf_file;
+        aconf->ebpf_t_config.flags |= EBPF_SOCKET_FILTER;
+    }
+
```

Comment:
fixed in https://github.com/OISF/suricata/pull/3895/commits/674e7c75e28150472da5c7063dfcc02f3b1d7539, please squash (part of that) in here